Sources/Sentry/SentryPerformanceTracker.m

```diff
@@ -0,0 +1,182 @@
+#import "SentryPerformanceTracker.h"
+#import "SentryLog.h"
+#import "SentrySDK+Private.h"
+#import "SentrySpan.h"
+#import "SentrySpanProtocol.h"
+#import "SentryTracer.h"
+
+static NSString *const SENTRY_PERFORMANCE_TRACKER_SPANS = @"SENTRY_PERFORMANCE_TRACKER_SPANS";
+static NSString *const SENTRY_PERFORMANCE_TRACKER_ACTIVE_STACK
+    = @"SENTRY_PERFORMANCE_TRACKER_ACTIVE_STACK";
+
+/*
+ * Auxiliary class to store tracking information.
+ */
+@interface SentrySpanTracker : NSObject
```

Comment:
`h`: `SentryTracer` already stores all spans. This seems a bit duplicate. Why do we need this class?

Reply:
`m`: Can we please add extensive tests for this class before merging?